use-cases/vehicleiot-uc/full-demo/service.yaml

```diff
@@ -114,6 +114,9 @@ topology_template:
         security_groups:  { get_input: security-groups } 
         flavor:           { get_input: flavor-name }
         username: centos
+        meta:
```

Comment:
Not sure this is needed, as #8  already added that metadata to sodalite.nodes.OpenStack.VM.